contracts/ccip_token_transfer/LinkiSwapMultipleFeeTokenSender.sol

```diff
@@ -32,12 +33,18 @@ contract LinkiSwapMultipleFeeTokenTransferor is OwnerIsCreator {
 
     IERC20 private s_linkToken;
 
+    AggregatorV3Interface internal dataFeed;
+
     /// @notice Constructor initializes the contract with the router address.
     /// @param _router The address of the router contract.
     /// @param _link The address of the link contract.
     constructor(address _router, address _link) {
         s_router = IRouterClient(_router);
         s_linkToken = IERC20(_link);
+
+        // BTC/ETH price feed contract address
+        dataFeed = AggregatorV3Interface(0x5fb1616F78dA7aFC9FF79e0371741a747D2a7F22);
```

Comment:
Please add feed contract for all the pairs currently on testnet.

Reply:
On Testnet, I think it's just BTC to ETH that is available for crypto-to-crypto pairs, others are Crypto to USD. Except you want me to include the Mainnet assets which includes several crypto to crypto pairs.?
![Uploading Screenshot 2023-12-10 at 04.15.29.png…]()